.env.sample

```diff
@@ -5,4 +5,6 @@ POSTGRES_DB=db
 POSTGRES_USER=user
 POSTGRES_PASSWORD=password
 POSTGRES_HOST=host
-POSTGRES_PORT=5432
\ No newline at end of file
+POSTGRES_PORT=5432
```

Comment:
its ok that we have real .env info here?

Reply:
My bad, thx)
I`ll fix that